docs/index.md

```diff
@@ -734,6 +734,32 @@ describe('add()', function() {
 });
 ```
 
+With `top-level await` you can collect your test data in a dynamic and asynchronous way while the test file is being loaded:
+
+```js
+// testfile.mjs
+// top-level await: Node >= v14.8.0 with ESM test file
```

Comment:
How about we provide full example to users can test it by copying this example.

```
import assert from 'assert';

function add(args) {
  return args.reduce((a, b) => a + b);
}
```

Reply:
done